modules/microshift-kickstart-rpm-install.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// microshift/microshift-kickstart-prep.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-kickstart-rpm-install_{context}"]
+= Creating a Kickstart file for a {microshift-short} RPM installation
+
+Use the Kickstart file provided with {microshift-short} to provision an RPM-based virtual machine.
+
+.Prerequisites
+
+* You set up the {microshift-short} Kickstart file.
+* You set up the [Activation Keys](https://console.redhat.com/insights/connector/activation-keys) and organization ID to activate your Red Hat subscription.
```

Comment:
I'm not sure if "set up" is appropriate. Generally, one needs to "know" these keys to proceed.